### PR TITLE
Fixed GIT_COMMIT_SHA for TestFlight build uploads.

### PR DESCRIPTION
**Pull request by neophit**
Modified setBuildVersion.sh to work with the AdHocDistribution configuration which is used to Archive and upload builds to TestFlight.

### CODE REVIEW

**Comment by bbodenmiller** [comment]
@neophit i'm getting the following error after cloning develop branch, any idea how to fix it?

image

**Comment by bbodenmiller** [comment]
@aaronbrethorst any ideas on how to fix my above build error?

File: scripts/setBuildVersion.sh [hunk 1]
```diff
@@ -1,16 +1,30 @@
...
+    gitCommitSHA=$(git rev-list HEAD | wc -l | tr -d ' ')
+fi
+
+#
+if [ -n "$gitCommitSHA" ]
+then
+    # Set bundle build version
     echo "#define GIT_COMMIT_SHA $gitCommitSHA" >> $PROJECT_TEMP_DIR/infoplist.prefix
+    # Tell Xcode to preprocess Info.plist
+    touch $PROJECT_DIR/Info.plist
```
Comment (aaronbrethorst) [inline scripts/setBuildVersion.sh]: Try wrapping this in quotes, i.e. `touch "$PROJECT_DIR/Info.plist"`

Comment (aaronbrethorst) [inline scripts/setBuildVersion.sh]: @bbodenmiller see https://github.com/OneBusAway/onebusaway-iphone/pull/259#discussion_r25144812 — the error suggests some path isn't properly escaping the space in your disk's name.

**Pull request merged**